csm/core/services/usl.py

```diff
@@ -438,93 +298,56 @@ async def post_register_device(
             reason = 'UDS base URL is not valid'
             Log.error(reason)
             raise CsmInternalError(desc=reason)
-        # Let ``_initialize_lyve_pilot_s3_resources()`` propagate its exceptions
-        (lyve_pilot_s3_credentials, lyve_pilot_s3_account, lyve_pilot_iam_user,
-         lyve_pilot_bucket) = await self._initialize_lyve_pilot_s3_resources(
-            s3_account_service,
-            s3_iam_users_service,
-            s3_bucket_service,
-            s3_account_name,
-            s3_account_email,
-            s3_account_password,
-            iam_user_name,
-            iam_user_password,
-            bucket_name,
-        )
-        try:
-            register_device_params = self._format_lyve_pilot_register_device_params(
-                url, pin, self._token)
-            access_params = await self._format_lyve_pilot_access_params(
-                lyve_pilot_s3_account,
-                lyve_pilot_iam_user,
-            )
-            registration_body = {
-                'registerDeviceParams': register_device_params,
-                'accessParams': access_params,
-            }
-        except Exception as e:
-            await self._cleanup(
-                s3_account_service,
-                s3_iam_users_service,
-                s3_bucket_service,
-                lyve_pilot_s3_credentials,
-                lyve_pilot_s3_account.get('account_name'),
-                lyve_pilot_iam_user.get('user_name'),
-                lyve_pilot_bucket.get('bucket_name'),
-            )
-            reason = 'Error on registration body assembly'
-            Log.error(f'{reason}---{str(e)}')
-            raise CsmInternalError(desc=reason)
+        # TODO validate registration body
```

Comment:
Is TODO done?

Reply:
The validation is being done by the view and by UDS. I have updated this comment with a more apt description.